pkg/session/fake/fake_session.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package fake implements a fake Session for testing.
+package fake
+
+import (
+	"context"
+
+	"github.com/vmware/govmomi/simulator"
+
+	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/session"
+	"sigs.k8s.io/cluster-api-provider-vsphere/test/helpers/vcsim"
+)
+
+const (
+	DefaultDataCenter = "*"
+)
+
+var (
+	Server   = ""
+	Username = ""
+	Password = ""
+)
+
+func MockDefaultSession() (*session.Session, error) {
+	return MockSession(DefaultDataCenter)
+}
+
+// MockSession GetOrCreate gets a cached session or creates a new one if one does not.
+func MockSession(dc string) (*session.Session, error) {
+	model := simulator.VPX()
+	model.Cluster = 1
```

Comment:
The model.Cluster returned by VPX() is already `1`.

Reply:
Noted, I will remove this statement.